src/content/docs/en/guides/content-collections.mdx

```diff
@@ -521,7 +521,7 @@ This guide shows you how to convert an existing Astro project with Markdown file
 
     The tutorial blog project also dynamically generates a page for each tag. This page now becomes:
 
-    ```js title="src/pages/tags/[tag].astro" "post.data" "getCollection(\"posts\")"
+    ```js title="src/pages/tags/[tag].astro" "post.data" "getCollection(\"posts\")" "post.data.title"
```

Comment:
We should be using the `astro` language here
````suggestion
    ```astro title="src/pages/tags/[tag].astro" "post.data" "getCollection(\"posts\")" "post.data.title"
````